docs/builders/iso.mdx

```diff
@@ -410,6 +410,62 @@ Here is a basic example creating a Fedora 29 server image with a Kickstart
 file served with Packer's HTTP server. Note that the iso file needs to be
 manually downloaded.
 
+<Tabs>
+<Tab heading="HCL2">
+
+```hcl
+variable "password" {
+  type    = string
+  default = "supersecret"
+}
+
+variable "username" {
+  type    = string
+  default = "apiuser@pve"
+}
+
+source "proxmox-iso" "fedora-kickstart" {
+  boot_command = ["<up><tab> ip=dhcp inst.cmdline inst.ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/ks.cfg<enter>"]
+  boot_wait    = "10s"
+  disks {
+    disk_size         = "5G"
+    storage_pool      = "local-lvm"
+    storage_pool_type = "lvm"
+    type              = "scsi"
+  }
+  efi_config {
+    efi_storage_pool  = "local-lvm"
+    efi_type          = "4m"
+    pre_enrolled_keys = true
+  }
+  http_directory           = "config"
+  insecure_skip_tls_verify = true
+  iso_file                 = "local:iso/Fedora-Server-dvd-x86_64-29-1.2.iso"
+  network_adapters {
+    bridge = "vmbr0"
+    model  = "virtio"
+  }
+  node                 = "my-proxmox"
+  password             = "${var.password}"
+  proxmox_url          = "https://my-proxmox.my-domain:8006/api2/json"
+  ssh_password         = "packer"
+  ssh_timeout          = "15m"
+  ssh_username         = "root"
+  template_description = "Fedora 29-1.2, generated on ${timestamp()}"
+  template_name        = "fedora-29"
+  unmount_iso          = true
+  username             = "${var.username}"
+}
+
+build {
+  sources = ["source.proxmox-iso.fedora-kickstart"]
+
```

Comment:
Out of consistency I'd suggest we remove this extra empty line since none of the other blocks have this

Reply:
Missed that line - done!